### ISSUE TITLE
Exclude hosts from errata installation

### ISSUE THREAD

**Issue by AnonJo** [stdevel/katprep#38]
It would be nice to have the option to exclude specific hosts from errata installation via part of the hostname.

**Comment by stdevel** [comment]
Hey AnonJo!

`katprep_maintenance` offers such a parameter:
```
$ katprep_maintenance -h
...
Foreman arguments:
...
  -E NAME, --exclude NAME
                        excludes particular hosts (default: no)
```

Is this what you mean?

**Comment by stdevel** [comment]
Hey AnonJo,
finally understood what you're talking about. You very talking about **parts** of the hostname. Currently, it is necessary to supply the *full* hostname.

Investigating in branch `feature-exclude_hostname-issue38`.

**Comment by stdevel** [comment]
Fixed in commit b28d0a75e2cf33bed9b0b856f795b57b72f9c13b.

You may now specify parts of the hostname as wildcard, e.g.:
```
katprep_maintenance ... -E stankowic.loc -E myhost
```

Specifying wildcard characters like * or % is not required.
